pybossa/jobs.py

```diff
@@ -820,6 +821,7 @@ def import_tasks(project_id, current_user_fullname, from_auto=False, **form_data
 def export_tasks(current_user_email_addr, short_name,
                  ty, expanded, filetype, filters=None, disclose_gold=False):
     """Export tasks/taskruns from a project."""
+    current_app.logger.info("exporting tasks...")
```

Comment:
This can be moved after line [829](https://github.com/bloomberg/pybossa/blob/a31dbf9a6c3b42ba6139eb647f7467f87f50ae79/pybossa/jobs.py#L829), when we've project info and include project_id into the logs
```python
f"exporting tasks for project {project.id}"
```